src/common/components/Logo.tsx

```diff
@@ -0,0 +1,19 @@
+import { AfterclassIcon } from "./CustomIcon/AfterclassIcon";
+import { AfterclassText } from "./CustomIcon/AfterclassText";
+
+type TAfterclassIconProp = {
+  hideText?: boolean;
+  hideLogo?: boolean;
+};
```

Comment:
Lets try to keep to using `interface` for simple props, unless we need more advanced features of `type`. Despite them being almost similar, I think it could make it easier.

Also, I'm not too sure if it'll be useful in our case to add `T` or `I` prefixes to props. I think we can keep it simple and just do `LogoProps`

```suggestion
export interface LogoProps {
  hideText?: boolean
  hideLogo?: boolean
}
```